Source/Scene/ModelExperimental/CustomShaderPipelineStage.js

```diff
@@ -384,41 +391,78 @@ function generateFragmentShaderLines(
   var positionBuiltins = generatePositionBuiltins(customShader);
 
   fragmentLines.enabled = true;
-  fragmentLines.attributeFields = attributeFields;
-  fragmentLines.fragmentInputFields = positionBuiltins.fragmentInputFields;
+  fragmentLines.attributeFields = attributeFields.concat(
+    positionBuiltins.attributeFields
+  );
   fragmentLines.initializationLines = positionBuiltins.initializationLines.concat(
     initializationLines
   );
 }
 
-function partitionAttributes(primitiveAttributes, shaderAttributeSet) {
+// computed positions should be treated like regular attributes for
```

Comment:
for...?

Reply:
ah this line is outdated anyway, I'll update it.